pkg/controller/certificaterequests/sync.go

```diff
@@ -216,7 +194,7 @@ func (c *Controller) updateCertificateRequestStatus(ctx context.Context, old, ne
 }
 
 // issuerKind returns the kind of issuer for a certificaterequest
-func issuerKind(cr *v1alpha1.CertificateRequest) string {
+func IssuerKind(cr *v1alpha1.CertificateRequest) string {
```

Comment:
Can this function move to `pkg/api/util` and be updated to accept a `v1alpha1.ObjectReference`?

Reply:
:+1: